LibraryApp/src/app/screens/BookDetails/components/CommentsList/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { View, FlatList } from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+import Comment from './components/Comment';
+
+class CommentsList extends Component {
+  renderItem = ({ item }) => {
```

Comment:
you dont need braces here `{`

Reply:
i really need it :( else i need to pass item.item to Comment component